checker.c

```diff
@@ -1,21 +1,26 @@
 #include <stdio.h>
 #include <assert.h>
-
-int batteryIsOk(float temperature, float soc, float chargeRate) {
-  if(temperature < 0 || temperature > 45) {
-    printf("Temperature out of range!\n");
-    return 0;
-  } else if(soc < 20 || soc > 80) {
-    printf("State of Charge out of range!\n");
-    return 0;
-  } else if(chargeRate > 0.8) {
-    printf("Charge Rate out of range!\n");
+#include "parameter_check.h"
+int temperature_status=0, soc_Status=0,charge_status =0,battery_status=0;
+int battery_parametertest(float para1,float para2,float para3) 
```

Comment:
To see if this function works, you would look at the console output, which says `Battery status is good condition`
Why not call this function also as `batteryIsInGoodCondition`?